examples/signals/web-bundle/signals.html

```diff
@@ -0,0 +1,84 @@
+<!--
+ If not stated otherwise in this file or this component's LICENSE file the
+ following copyright and licenses apply:
+
+ Copyright 2020 RDK Management
+
+ Licensed under the Apache License, Version 2.0 (the License);
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+
+    <script src="../../../dist/lightning-web.js"></script>
+    <script src="../../../devtools/lightning-inspect.js"></script>
+</head>
+<body>
+    <script type="module">
+        //attachInspector(lng)
+
+        window.onload = function() {
+            class Box extends lng.Component {
+                static _template() {
+                    return {
+                        w: 300,
+                        h: 300,
+                        rect: true,
+                        color: 0xffffff00,
+                    }
+                }
+
+                _init() {
+                    this.signal('signalHandler1')
+                    this.signal('signalHandler2')
+                }
+            }
+
+            class BasicUsageExample extends lng.Application {
+                static _template(ctx) {
+                    return {
+                        w: 900,
+                        h: 900,
+                        Bg: {
+                            src: "../../landscape.jpg", scale: 1,
+                        },
+                        Box: {
+                            type: Box,
+                            x: w => w / 2,
+                            y: h => h / 2,
+                            mount: 0.5,
+                            signals: {
+                                signalHandler1: ctx.signalHandler1,
```

Comment:
`true` and `string` handlers could be added to example (see [docs](https://rdkcentral.github.io/Lightning/docs/components/communication/signal#docsNav))